### PR TITLE
[Shevchyshen Petro]HomeWork. Lessone 1. Resume

### PR DESCRIPTION
**Pull request by PetroShevchyshen**

### CODE REVIEW

File: home/Shevchyshen-Petro/Lessone1/HomeWork1(en).html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <div>
+            
+            <button><a href="/HomeWork1.html">UKR</a></button>
+    
+            <h1 style="text-align: center;border-bottom: 2px solid #d8d8d8;">Resume</h1>
+    
+            <h2 style="text-align: center;">Shevchyshen Petro</h2>
+    
+                <div>
+                    <img src="/img/IMG_0527.JPG" alt="Petro" style="width: 300px;">
```
Comment (avatar1807) [inline home/Shevchyshen-Petro/Lessone1/HomeWork1(en).html]: The wrong photo link will not work

File: home/Shevchyshen-Petro/Lessone1/HomeWork1.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <div>
+        
+            <button><a href="/HomeWork1(en).html">EN</a></button>
+
+            <h1 style="text-align: center; border-bottom: 2px solid #d8d8d8;">Резюме</h1>
+
+            <h2 style="text-align: center;">Шевчишен Петро Петрович</h2>
+
+                <div>
+                    <img src="/img/IMG_0527.JPG" alt="Petro" style="width: 300px;">
```
Comment (avatar1807) [inline home/Shevchyshen-Petro/Lessone1/HomeWork1.html]: The wrong photo link will not work

Comment (PetroShevchyshen) [reply home/Shevchyshen-Petro/Lessone1/HomeWork1.html]: Fix

File: home/Shevchyshen-Petro/Lessone1/HomeWork1(en).html (hunk 1 above)
Comment (PetroShevchyshen) [reply home/Shevchyshen-Petro/Lessone1/HomeWork1(en).html]: Fix

**Pull request merged**